2016-06-03.md

```diff
@@ -0,0 +1,34 @@
+# Improving Rust, Letting Go of Python, Open Finances, and More
+
+## Multiple language support
+
+### Ramping up Rust
+We've long been intrigued by the Rust Language, and have lately been pushing forward for a more native Rust support experience, including easy deploy from the `t2` command line tool.
```

Comment:
I might suggest changing "Rust Language" to "Rust Programming Language" (since that's how they refer to the project) and add a link to https://www.rust-lang.org/.